ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -113,8 +113,9 @@ export default Route.extend({
         .catch((err) => {
           // if we're at the root we don't want to throw
           if (backendModel && err.httpStatus === 404 && secret === '') {
+            this.set('noMetadataPermissions', false);
```

Comment:
If it's false by default this might be redundant

Reply:
It's true by default. I just tested without and I get the original error.

Reply:
https://github.com/hashicorp/vault/pull/14276/files/ca89590cacc2fe75575ed457c97d8c2e78ad63a7#diff-b30928193c4fb5f7951070126947426c653b4ee983ac47cb8c3d9c57c8cf9165R14